test/view/sldview/sdlview.cpp

```diff
@@ -0,0 +1,36 @@
+#include "../../../src/view/sdlview/sdlview.h"
+#include <catch2/catch.hpp>
+#include <exception>
+
+TEST_CASE("SDLView.getRelativePosition can calculate the correct position on the screen") {
+    SDLView sdlview = SDLView();
+
+    const int windowWidth = 1080;
+    const int windowHeight = 1080;
+    const double worldWidth = 1000;
+    const double worldHeight = 1000;
+
+    SECTION("The conversion works successfully at minimum position") {
+        Point position = {0,0};
+        Point expectedPosition = {0,0};
+        REQUIRE(
+                sdlview.getRelativePosition(position, worldWidth, worldHeight, windowWidth, windowHeight) == expectedPosition
+                );
+    }
+
+    SECTION("The conversion works successfully at maximum position") {
+        Point position = {1000,1000};
+        Point expectedPosition = {1080,1080};
```

Comment:
you can use an even shorter notation:
```Point position (1000, 1000);```

Reply:
I mean it's not much shorter but I guess I can use it

Reply:
you don't have to but knowing that it also works is good